.github/workflows/sqlachemy_publish.yml

```diff
@@ -0,0 +1,33 @@
+name: Push SQLAlchemy Docker Image
+
+on:
+  pull_request:
+  create:
+    tags:
+      - v*
+
+jobs:
+  push-github:
+    name: Push to GHCR
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: plugins
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: "0"
+      - name: Push SQLAlchemy Image to GitHub Registry
+        uses: whoan/docker-build-with-cache-action@v5
+        with:
+          # https://docs.github.com/en/packages/learn-github-packages/publishing-a-package
+          username: "${{ secrets.FLYTE_BOT_USERNAME }}"
+          password: "${{ secrets.FLYTE_BOT_PAT }}"
+          image_name: ${{ github.repository_owner }}/flytekit
+          image_tag: sqlalchemy-v0.20.1,sqlalchemy-${{ github.sha }},sqlalchemy-${{ github.event.ref }}
```

Comment:
Sorry i missed this.  we shouldn't hardcode the version here.  We don't want to accidentally override the existing image if we forget to change this value.  The release sha should be good enough right?